src/main/resources/application.properties

```diff
@@ -17,7 +17,15 @@ alert.accumulator.cron=0 0/1 * 1/1 * *
 alert.digest.daily.cron=0 0 0 1/1 * ?
 alert.digest.realtime.cron=0 0/1 * 1/1 * *
```

Comment:
If we are going to use this file, we should remove any fields that need to be specified by the user through the UI.

Reply:
That's a good point. We won't need half this file anymore because of the UI. I'll delete